### ISSUE TITLE
"one-to-one authentication SHALL occur before a PIV Card is issued and activated

### ISSUE THREAD

**Issue by wszwerc** [usnistgov/FIPS201#389]
### All Fields Are Required
**Organization Name (N/A, if individual)**: DHS

**Organization Type (see below for codes)**: 1 - Federal
 
**Reference (Include section/paragraph or pdf line number)**: Line 835-838
 
**Comment (Include rationale for comment)**: "Before the PIV Card is provided to the applicant, the issuer SHALL perform a one-to-one comparison of the applicant against biometric data records available on the PIV Card or in the PIV enrollment record."

As the language stands, it would seem that it is okay to perform a one-to-one biometric authentication after the card is printed and activated, but before an issuer hands the card (or ships the card) to an individual. 

**Suggested Change**: Recommend that the language be updated to explicitly require that a one-to-one authentication SHALL occur before a PIV Card is issued and activated, to ensure it is the same identity being issued to, before any changes occur to the card or identity record. 
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

**Comment by hferraio** [comment]
discuss.  The goal of the 1:1  biometric match is to ensure that the person receiving the card is the actual cardholder.  This is done just before handing the card to the cardholder.  It is not meant to ship an activated card.  Maybe we need to state this?

 The suggested change  is too restrictive and not  backward compatible with install base (DoD-11 in [FIPS 201-2 comments ] (https://csrc.nist.gov/CSRC/media/Publications/fips/201/2/final/documents/fips201_2_2012_draft_comments_and_dispositions.pdf).

**Comment by regenscheid** [comment]
Will conduct additional research to revise this language- either accepting the proposed language or investigating how the biometric match would be done against the templates on the new card.

**Comment by gfiumara** [comment]
Decline. The suggestion would invalidate current implementation and there are safeguards in place (will be provided ONLY after successful biometric comparison)
